Added server start-up routine suitable for Windows x64 environments

danmenzies-jerram opened the pull request.

danmenzies committed:
Added server start-up routine suitable for Windows x64 environments

DesignyourCode reviewed (changes requested):
For consistency across platforms, it would be better to keep this as one command. Otherwise you run  the risk of duplicating commands in the `package.json`.

I would recommend looking at installing this package, and setting this up to work with the existing server command: https://www.npmjs.com/package/cross-env

danmenzies-jerram commented:
Good call, @DesignyourCode. I'll take a look :-)

danmenzies committed:
Pulled current version of master into this bug/windows-launch

danmenzies committed:
Added cross-env, so x64 Windows Dev Machines can launch the server without errors

danmenzies committed:
Restored test/index.html to default, after accidental commit

danmenzies committed:
Updated NPM Package list

danmenzies-jerram commented:
Added the [cross-env package](https://www.npmjs.com/package/cross-env), @DesignyourCode, and updated the scripts in the `package.json` file to reflect the new config. Works a treat :-)

Would you mind reviewing again?
